grakn-test/pom.xml

```diff
@@ -198,6 +197,23 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19.1</version>
+                <configuration>
+                    <systemProperties>
+                        <logback.configurationFile>${main.basedir}/conf/test/logback-test.xml</logback.configurationFile>
```

Comment:
This is how we specify the logback file in the other poms: 
`<logback.configurationFile>../conf/test/logback-test.xml</logback.configurationFile>`